.pre-commit-config.yaml

```diff
@@ -91,3 +91,33 @@ repos:
     - id: add-license-headers
       args:
       - --start_year=2021
+
+- repo: local
+  hooks:
+    - id: check-vulnerabilities-dependencies
+      name: Check Vulnerabilities Dependencies
+      entry: python -c "import os; import subprocess; subprocess.run(['python', 'requirements/requirements_vulnernabilities.py'])"
+      language: python
+      types: [python]
+      pass_filenames: false
+      always_run: true
+
+    - id: check-vulnerabilities
```

Comment:
This hook performs actual vulnerability check and reports them if there are any. In the description of this PR we can see the actual reporting of vulnerabilities.